notebooks/curation_api/python/get_collections.ipynb

```diff
@@ -89,19 +113,30 @@
   {
    "cell_type": "markdown",
    "id": "6085d322",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "### Fetch Collection"
+    "### Fetch Collection\n",
+    "#### arguments:\n",
+    "visibility: \"PRIVATE\" to retrive private collections\n",
```

Comment:
not sure why, but the github rendering isn't recognizing this newline. as a result, the args are appearing as part of the same line, which is confusing.

Reply:
it rendered fine on my system, but to be safe I made them bullet points.